<launch>
    
    <node pkg="help_carry_shopping_bag" type="select_cmd_vel.py" name="select_cmd_vel_node" />
		<node pkg="help_carry_shopping_bag" type="navigation.py" name="go_and_back_nav" output="screen"/>
    <node pkg="help_carry_shopping_bag" type="send_stop_cmd_vel" name="send_stop_cmd_vel_node" />
    <node pkg="help_carry_shopping_bag" type="timeout.py" name="Timeout" />
    <node pkg="topic_tools" type="mux" name="cmd_vel_mux" args="cmd_vel stop_cmd_vel follow_cmd_vel nav_cmd_vel" output="screen" />
    <node pkg="sound_play" type="soundplay_node.py" name="soundplay_node" />
		<node pkg="help_carry_shopping_bag" type="help_carry.py" name="car_help_recognizer" />

</launch>
